docs/formats/zng.md

```diff
@@ -135,8 +135,8 @@ Values for the `format` byte are defined in the
 
 > This arrangement of message blocks separating types and values allows
 > for efficient scanning and parallelization.  In general, values depend
-> on type definitions but as long as all of the types are known by the
-> time values are used, decoding can be done in parallel.  Likewise, since
+> on type definitions but as long as all of the types are known before
+> values are used, decoding can be done in parallel.  Likewise, since
```

Comment:
I proposed this change because upon first read I had difficulty parsing the sentence, as my eyes got stuck on "time values" and I found myself wondering "Wait... what do the values of `time` types have to do with this?" Once I figured out how I was misinterpreting the sentence, I figured an alternate wording might be justified.